frontend/libs/client/feature-enrollment/src/lib/components/register/register.component.ts

```diff
@@ -118,6 +120,17 @@ export class RegisterComponent implements OnInit {
       .add(() => (this.loading = false));
   }
 
+  public loginAutomatically() {
+    this.loading = true;
+    this.userService
+      .login(this.registrationForm.controls.username.value, this.registrationForm.controls.password.value)
```

Comment:
The backend response contains a valid token in the http header. So you can use the provided token instead of logging in with username and password.